examples/falcon/hello.py

```diff
@@ -33,8 +60,68 @@ def on_post(self, req, resp, **kwargs):
 
 # things will handle all requests to the '/things' URL path
 app.add_route('/hello', hello)
+app.add_route('/test/html_response', HTML_test_resource())
+app.add_route('/test/xml_response', XML_test_resource())
+app.add_route('/test/json_response', Json_test_resource())
 
-# Add Moesif Middleware
-app = MoesifMiddleware(app, {
+def identify_user(app, environ):
+    return '12345'
+
+
+def identify_company(app, environ):
+    return '67890'
+
+
+def get_token(app, environ):
+    # If you don't want to use the standard WSGI session token,
+    # add your custom code that returns a string for session/API token
+    return "XXXXXXXXXXXXXX"
+
+
+def should_skip(app, environ):
+    # Your custom code that returns true to skip logging
+    return "health/probe" in environ.get('PATH_INFO', '')
+
+
+def get_metadata(app, environ):
+    metadata = None
+    try:
+        metadata = {
+            'response-body': environ['response-body'],
+            'Content-Type': environ['response-headers']['Content-Type'.lower()],
+            'Content-Length': environ['response-headers']['Content-Length'.lower()],
+            'X-Moesif-Transaction-Id': environ['response-headers']['X-Moesif-Transaction-Id'.lower()]
+        }
+    except KeyError:
+        print('environ has no field [response-body] or [response-headers]')
+    return metadata
+
+
+def mask_event(eventmodel):
+    # Your custom code to change or remove any sensitive fields
+    if 'password' in eventmodel.response.body:
+        eventmodel.response.body['password'] = None
+    return eventmodel
+
+moesif_settings = {
     'APPLICATION_ID': 'Your Moesif Application Id',
-})
+    'IDENTIFY_USER': identify_user,
+    'IDENTIFY_COMPANY': identify_company,
+    'LOG_BODY': True,
+    'SKIP': should_skip,
+    'DEBUG': False,
+    'MASK_EVENT_MODEL': mask_event,
+    'GET_SESSION_TOKEN': get_token,
+    'GET_METADATA': get_metadata,
+    'CAPTURE_OUTGOING_REQUESTS': False
+}
+
+# Add Moesif Middleware
+app = MoesifMiddleware(app, moesif_settings)
+
+if __name__ == '__main__':
+    with make_server('', 8000, app) as httpd:
```

Comment:
any reason to use `make_server` to start the server instead of `gunicorn`?

Reply:
I saw some examples are with `make_server`, let try `gunicorn`

Reply:
Changed